executor/master-client.go

```diff
@@ -2,10 +2,12 @@ package executor
 
 import (
 	"context"
-	"errors"
 	"time"
 
 	"github.com/hanfei1991/microcosom/pb"
+	"github.com/hanfei1991/microcosom/pkg/errors"
```

Comment:
the import path should be updated

Reply:
shouldn't it be "github.com/hanfei1991/microcosm/pkg/errors" ?